.changeset/serious-crews-shake.md

```diff
@@ -0,0 +1,27 @@
+---
+"@thirdweb-dev/wallets": minor
+---
+
+feat: add phone number verification to embedded wallets
```

Comment:
human readable changeset please, more like what we have for the `@thirdweb-dev/react` below